app/src/main/res/layout/activity_main.xml

```diff
@@ -11,7 +11,7 @@
     <include layout="@layout/layout_toolbar" />
 
     <FrameLayout
-        android:id="@+id/controller_container"
+        android:id="@+id/controllerContainer"
```

Comment:
I'm not sure about the camel case in android resource id's because of this. I found some related disscussion on reddit https://www.reddit.com/r/androiddev/comments/30vy3y/how_do_you_name_your_view_ids/